packages/flutter_tools/lib/src/commands/create_base.dart

```diff
@@ -100,7 +100,8 @@ abstract class CreateBase extends FlutterCommand {
       abbr: 'i',
       defaultsTo: 'swift',
       allowed: <String>['objc', 'swift'],
-      help: 'The language to use for iOS-specific code, either Objective-C (legacy) or Swift (recommended).'
+      help: '(deprecated) The language to use for iOS-specific code, either Objective-C (legacy) or Swift (recommended).',
```

Comment:
Optional: maybe we should swap the order so the recommended option is first?

Reply:
Good idea.  Did the same for `android-language`.